### ISSUE TITLE
New environment for IA refresh

### ISSUE THREAD

**Issue by phildexter** [mozilla/foundation.mozilla.org#4671]
tldr; this is for a new test environment of foundation.mozilla.org.uk to help test the IA and content changes.

**Background**
Currently there are two environments. Live hosts the production content. Staging is used to test changes before they are pushed to production. Staging is refreshed with Live data once per week to ensure the tests are representative. Because any data entered on Staging is lost during the refresh, it’s not feasible to test large scale changes to content of menu structures in this environment.

To facilitate the gathering and testing of content changes for this project, we will setup a new environment called IA Test. This will get a one-off copy of production data, but will not be synced weekly after this. This allows the design and content teams to make edits to the site over a few weeks, and get sign off of content in-situ, on the sites pages.

Once the new content and IA has been signed off for release to production, we cannot simply switch DNS to this environment as the content will be out of date (having been changed on production since the cut was taken). 

Instead we will plan for a content freeze on production for a few(?) days. At this point (either) Staging (or another replica environment) will have all the latest content. We will then need to alter the production content to make it look like the new IA Test does.

Alternative - use https://github.com/wagtail/wagtail-transfer

Needs an estimate from @cadecairos

**Comment by patjouk** [comment]
This website is ready at this address: https://ia-refresh.mofostaging.net
People can use their google account to log in.

I'm opening a follow-up ticket with everything that will need to be removed when we delete this app.

**Comment by phildexter** [comment]
Great thanks! I can access the FE at least - will look to @taisdesouzalessa and @kristinashu to test the login to the admin.

**Comment by phildexter** [comment]
Yes - we will have separate tickets to build the homepage once we have final designs. So for now, we could probably ignore the home page, but other pages should be possible to change.
